girder/api/v1/response.py

```diff
@@ -134,9 +139,13 @@ def createResponseItem(self, subject_id, metadata):
                 ) == str
             ) else "[Unknown Applet]",
             reuseExisting=True, public=False)
+        # TODO: fix above [Unknown Applet]. Let's pass an appletName
```

Comment:
- This folder name is an internal thing, not anything anyone should ever have to see or care about. `appletName` instead of `appletId` is fine with me, though. (#105)
- "[Unknown Applet]" catches, as the name suggests, responses to unknown applets. @akeshavan, do you think we should raise an exception instead?

Reply:
Maybe we should send appletName or appletId as a parameter, then? and throw an error if its not provided